scipy/stats/_continuous_distns.py

```diff
@@ -2957,6 +2957,206 @@ def _entropy(self, c):
 genhalflogistic = genhalflogistic_gen(a=0.0, name='genhalflogistic')
 
 
+class genhyperbolic_gen(rv_continuous):
+    r"""A generalized hyperbolic continuous random variable.
+
+    %(before_notes)s
+
+    See Also
+    --------
+    t, norminvgauss, geninvgauss, laplace, cauchy
+
+    Notes
+    -----
+    The probability density function for `genhyperbolic` is:
+
+    .. math::
+
+        f(x, p, a, b) =
+            \frac{(a^2 - b^2)^{p/2}}
+            {\sqrt{2\pi}a^{p-0.5}
+            K_p\Big(\sqrt{a^2 - b^2}\Big)}
+            e^{bx} \times \frac{K_{p - 1/2}
+            (a \sqrt{1 + x^2})}
+            {(\sqrt{1 + x^2})^{1/2 - p}}
```

Comment:
Readability: define named variables for `p-1/2`, `a^2-b^2` etc. Also prefer X / Y to \frac{X}{Y}

Reply:
Thanks @ev-br but doc builds are not completing reliably these days, so I would rather not ask @gbonomib to tweak that right now.  I will submit a followup PR immediately after this merges and ask for your review.